### DISCUSSION TITLE
Would ddns-updater watch config file changes?

### DISCUSSION THREAD

**Discussion by shizunge** [qdm12/ddns-updater#340] [Q&A] [answered]
Would ddns-updater watch config file changes?
Or it only loads config file at beginning?

**Comment by qdm12** [comment, marked as answer]
Just at the progam start for the time being. Subscribe to #372 for this new feature.

**Comment by kattivius** [comment]
Hi,
Just to make sure is clear for me...
If I add new subdomain after the container (I have it in a docker) is started, anything added will not be taken in consideration beside the initial domains/subdomains defined when last started the container?
If so, the only way would be to restart the container for each modification, correct?

**Reply by qdm12** [reply]
Correct! 👍
